src/index.js

```diff
@@ -1,15 +1,17 @@
-import {UICorePlugin, Events} from 'clappr'
-import $ from 'jQuery'
-import './style.sass'
+import {UICorePlugin, Styler, Events, template} from 'clappr'
+import pluginHtml from './public/scrub-thumbnails.html'
+import pluginStyle from './public/style.sass'
 
 export default class ScrubThumbnailsPlugin extends UICorePlugin {
   get name() { return 'scrub-thumbnails' }
 
   get attributes() {
     return {
-      'class': this.name
+      'class': this.name,
+      'data-scrub-thumbnails': ''
```

Comment:
Follow other plugins as an example and added other names in the tags to facilitate debug in IE11. But this is not necessary